.github/CONTRIBUTING.md

```diff
@@ -39,42 +56,45 @@ Sometimes, Discord doesn't give the token at first. All you have to do is;
 #### What **Bot Permissions** should I choose in the **URL Generator** section?
 After you choose `applications.command` and `bot` as your scopes, a new **Bot Permissions** section will be available below. Just as before, toggle the **Administrator** checkbox.
 
-### Issues
+## Setting Up Development Environment
 
-#### Create a new issue
+1. Clone the repo so that you can make your changes without affecting the original project until you're ready to merge them.
 
-If you spot a problem with BICS-BOT, [search if an issue already exists](https://docs.github.com/en/github/searching-for-information-on-github/searching-on-github/searching-issues-and-pull-requests#search-by-the-title-body-or-comments). If a related issue doesn't exist, you can open a new issue using a relevant [issue form](https://github.com/Luxembourg-Open-Source-Club/BICS-BOT/issues/new/choose).
+2. [Create a virtual environment](https://docs.python.org/3/library/venv.html#creating-virtual-environments) for Python, and [activate it](https://docs.python.org/3/library/venv.html#how-venvs-work). _**If you are on Linux**_, you must first install the package `python3-venv` using your distro's package manager.
 
-#### Solve an issue
+3. Install requirements via `$ pip install -r requirements.txt`
 
-Scan through our [existing issues](https://github.com/Luxembourg-Open-Source-Club/BICS-BOT/issues) to find one that interests you. You can narrow down the search using `labels` as filters. Once you spot an issue you are interested in solving, feel free to self-assign it and create a branch for it to start working. Pay attention to issues labeled **good first issue**
+4. Create a file `.env` in the root directory, and add your test bot's token by entering this in the file: `TOKEN_BOT_CLONE=<insert your token>`
 
-### Make Changes
+5. Create a working branch and start with your changes!
 
-#### Make changes locally
+### Running the Bot
 
```

Comment:
Here we should also had that before modifying any code, it is encouraged to run the bot first and make sure that it is running as expected, by also testing to send a command on the test server.